units/meson.build

```diff
@@ -94,6 +94,8 @@ units = [
          'sockets.target.wants/'],
         ['systemd-networkd.socket',             'ENABLE_NETWORKD',
          join_paths(pkgsysconfdir, 'system/sockets.target.wants/')],
+        ['systemd-resolved-update-resolvconf.path', '',],
+        ['systemd-resolved-update-resolvconf.service', '',],
```

Comment:
i'd prefer if we wouldn't have this bit in systemd. It's a Debianism, and should really live downstream I think.

That said, I think for systems not using Debian's resolvconf tool, this new stub file should become the default. Can you update tmpfiles.d/etc.conf.m4 accordingly?

Reply:
resolvconf stuff is dropped. tmpfiles updated.